changelog/fix_false_positives_for_style_arguments_forwarding_one.md

```diff
@@ -0,0 +1 @@
+* [#12087](https://github.com/rubocop/rubocop/issues/12087): Fix false positive for `Style/ArgumentsForwarding` with additional arg passed in send node. ([@owst][])
```

Comment:
I think changelog entries can be merged into this representative issue number entry.